webapp/components/test-run.html

```diff
@@ -159,7 +159,9 @@
       }
 
       sevenCharSHA(sha) {
-        return sha.substr(0, 7);
+        return sha.length > 7
```

Comment:
Doesn't this do the same as before, or what does substr do when asked to make a longer string?

Reply:
Reverted.